third-party/gasnet/README

```diff
@@ -18,9 +18,9 @@ The modifications that we have made to the official GASNet release are
 as follows:
 
 * Pulled in an upstream fix for mpi-spawner -N support for Open MPI 3.x
-   - https://bitbucket.org/berkeleylab/gasnet/commits/e9088bc72
+   - https://bitbucket.org/berkeleylab/gasnet-ex/commits/bdd352a5c
```

Comment:
These commit IDs don't match the ones given in the commit messages for the cherry-picks themselves (not our commit IDs, I wouldn't expect that, but they also don't match the commit IDs we say we're cherry-picking from).  Should they?

Reply:
For our https://github.com/chapel-lang/chapel/pull/11874/commits/52cad0e218f30eb3f7e38d4ccfc00537ddfb8a8f (which cherry-picked https://bitbucket.org/berkeleylab/gasnet-ex/commits/bdd352a5c) you're seeing something like: 

```
(cherry picked from commit e9088bc72825652a6221a778cbc5617911520f66)
```

That's from the EX commit message itself. They cherry picked from the GASNet-1 or dev branch onto EX branch, and then I cherry-picked it from the EX branch. (Note that that matches the value we used to cherry-pick too.)